src/types.ts

```diff
@@ -541,3 +551,10 @@ export interface IKVWalletIndex extends IKVStoreIndex<void> {
   cleanAccessData(): Promise<void>;
   cleanWalletData(clear: boolean): Promise<void>;
 }
+
+export interface INcData {
```

Comment:
I'm wondering if we should persist the history or let it be loaded every time the user load the wallet?

If we decide to persist, I'm not sure if we should add the history here or open a map just for it.